### DISCUSSION TITLE
Development Wiki needs updating

### DISCUSSION THREAD

**Discussion by breadoven** [EdgeTX/edgetx#2034] [General]
The development Wiki seems to be out of date at least for Ubuntu 20.04:

"make -j`nproc` companion28" doesn't work, "make -j`nproc` companion" does work though even though the final file is still called `companion28`.

`./companion28` and` ./simulator28` don't work from the build directory. They only work if you cd to the `"native" `directory in the build directory.

**Comment by raphaelcoeffic** [comment]
Thx, we should probably have a meta target without the version number, that should make it simpler.
